examples/login/README.md

```diff
@@ -0,0 +1,16 @@
+# Example script: log into a website with AgentQL
+
+This example demonstrates how to log into a website using AgentQL.
```

Comment:
Should we also add ability to save the session and reuse it for future queries instead of logging in?

Reply:
Yes! I think that's achievable through Playwright's native API. We could add an additional example showing this capacity.

Reply:
I was actually talking about this one. I.e. login in is not a super specific use case — it's just another "filling the form" example, but the "skipping the login if we have saved info" is smth that is valuable. (btw,, do we have another "form filling" example?